domains/solomon.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "Personal site",
+    "repo": "https://github.com/solomonsweb/newportfolio",
+    "owner": {
+        "username": "solomonsweb",
+        "email": "solomon@cymruwifi.co.uk"
+    },
+    "record": {
+        "CNAME": "solomonsweb.github.io/newportfolio"
```

Comment:
CNAME records cannot contain file paths. Provide that path URL in the PR template as preview, and use just the hostname.